recipe/run_conda_forge_build_setup_osx

```diff
@@ -39,6 +39,13 @@ if [ ! -z "$CONFIG" ]; then
     echo "CONDA_BUILD_SYSROOT:" >> ./.ci_support/${CONFIG}.yaml
     echo "- ${CONDA_BUILD_SYSROOT}" >> ./.ci_support/${CONFIG}.yaml
     echo "" >> ./.ci_support/${CONFIG}.yaml
+
+    if [ ! -z "$CI" ]; then
+        echo "CI:" >> ./.ci_support/${CONFIG}.yaml
```

Comment:
Maybe add a blank line before? Same for the rest as well.

Reply:
There's a blank line added in Line 41